Lib/test/test_uuid.py

```diff
@@ -311,6 +311,26 @@ def test_getnode(self):
         node2 = self.uuid.getnode()
         self.assertEqual(node1, node2, '%012x != %012x' % (node1, node2))
 
+    # bpo-32502: UUID1 requires a 48-bit identifier, but hardware identifiers
+    # need not necessarily be 48 bits (e.g., EUI-64).
+    def test_uuid1_eui64(self):
+        # Reset any cached node value
+        self.uuid._node = None
+
+        # Confirm that uuid.getnode ignores hardware addresses larger than 48
+        # bits
+        bad_getter = lambda: 1 << 48
+        node = self.uuid.getnode(getters=[bad_getter])
+        self.assertTrue(0 < node < (1 << 48), '%012x' % node)
```

Comment:
Okay, so why is that assertion going to be true?  The reason is that the random UUID generator is always appended to `getters`, and it's that latter one that will return a valid value.  I think you should write a comment above line 323 explaining that, otherwise it could be a bit of a head scratcher for someone just reading the test.

Reply:
Quite right. Hopefully my new comment explains this.